internal/cmd/schema-registry/command_cluster_update.go

```diff
@@ -102,35 +102,63 @@ func (c *command) getConfigUpdateRequest(cmd *cobra.Command) (srsdk.ConfigUpdate
 		return updateReq, err
 	}
 
-	var metadataDefaults srsdk.Metadata
-	var metadataOverrides srsdk.Metadata
-	var rulesetDefaults srsdk.RuleSet
-	var rulesetOverrides srsdk.RuleSet
-
-	if err := readPathFlag(cmd, "metadata-defaults", &metadataDefaults); err != nil {
-		return updateReq, err
+	updateReq = srsdk.ConfigUpdateRequest{
+		Compatibility:      strings.ToUpper(compatibility),
+		CompatibilityGroup: compatibilityGroup,
 	}
 
-	if err := readPathFlag(cmd, "metadata-overrides", &metadataOverrides); err != nil {
-		return updateReq, err
+	metadataDefaultsPath, err := cmd.Flags().GetString("metadata-defaults")
+	if err != nil {
+		return srsdk.ConfigUpdateRequest{}, err
+	}
+	if metadataDefaultsPath != "" {
+		var metadataDefaults srsdk.Metadata
+		err := read(metadataDefaultsPath, &metadataDefaults)
+		if err != nil {
+			return srsdk.ConfigUpdateRequest{}, err
+		}
+		updateReq.DefaultMetadata = &metadataDefaults
```

Comment:
```suggestion
		err := read(metadataDefaultsPath, updateReq.DefaultMetadata)
		if err != nil {
			return srsdk.ConfigUpdateRequest{}, err
		}
```

Reply:
I think you would need to add `updateReq.DefaultMetadata = new(srsdk.Metadata)` on the line before!

Reply:
Either way is fine, but this way doesn't create a new variable.